features/json-ld/context.feature

```diff
@@ -17,15 +17,15 @@ Feature: JSON-LD contexts generation
             "name": "http://schema.org/name",
             "alias": "https://schema.org/alternateName",
             "foo": "#Dummy/foo",
-            "dummyDate": "#Dummy/dummyDate",
-            "jsonData": "#Dummy\/jsonData",
+            "dummy_date": "#Dummy/dummy_date",
+            "json_data": "#Dummy\/json_data",
             "dummy": "#Dummy/dummy",
-            "relatedDummy": {
-                "@id": "#Dummy/relatedDummy",
+            "related_dummy": {
+                "@id": "#Dummy/related_dummy",
```

Comment:
They are not OK. The key should be changed but not the value because the  referenced id is not touched: https://github.com/ogizanagi/DunglasApiBundle/blob/vocab_context_name_converter/features/hydra/vocab.feature#L105